### ISSUE TITLE
More descriptive error emails to admins

### ISSUE THREAD

**Issue by kmahar** [haystack/murmur#85]
It would be useful to include more info in the admin email for an error. Right now the email is identical to the one the poster receives. The poster likely know which post was the offending one (unless if they are making several posts in a row) -- but as an admin one must go to the logs to figure out which list, post and user it was.

**Comment by amyxzhang** [comment]
yep. though right now most of those emails right now are because of the
list forwarding issue btw.

On Fri, Apr 22, 2016 at 10:02 PM, Kaitlin Mahar <notifications@github.com>
wrote:

> It would be useful to include more info in the admin email for an error.
> Right now the email is identical to the one the poster receives. The poster
> likely know which post was the offending one (unless if they are making
> several posts in a row) -- but as an admin one must go to the logs to
> figure out which list, post and user it was.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/haystack/murmur/issues/85>
>



-- 
Amy X. Zhang | Ph.D. student at MIT CSAIL | http://people.csail.mit.edu/axz
| @amyxzh
